lib/menulst.js

```diff
@@ -178,15 +178,24 @@ function buildMenuLst(opts, cb) {
         'min_mem64 1024'
     ];
 
+    var console_config = getConsoleConfig(c);
+
     /*
      * Intentionally simplified GRUB menu for Linux
      */
     if (opts.os === 'linux') {
```

Comment:
Dumb question --> Does your capitalization of Linux in sdc-designation cause problems here?

Reply:
That's not a dumb question.

In this case it does not, because this field is referring to the platform image OS which is not derived from `uname`. This one is referring to this:

```
[root@headnode (quartz) ~]# sdcadm platform list
VERSION           CURRENT_PLATFORM  BOOT_PLATFORM  LATEST  DEFAULT  OS
20210603T160822Z  1                 1              true    false    linux
20210128T022709Z  3                 3              false   true     smartos
```